scriptworker/monitoring/nagios_file_age_check.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+"""
+Checks for the existence and age of specific files, and that they are not
+too old.
+
+Allow 'old' to be defined as warning and critical arguments.
+
+You can either:
+- Specify one file to check
+- Give the path to a file, that contains a newline-separated list
+  of the files to check.
+
+This script will produce a multi-line nagios report in either case.
+
+"""
+import os
+import sys
+import time
+import argparse
+
+# Nagios plugin exit codes
+STATUS_CODE = {
+    'OK': 0,
+    'WARNING': 1,
+    'CRITICAL': 2,
+    'UNKNOWN': 3,
+}
+
+DEFAULT_WARNING = 45
+DEFAULT_CRITICAL = 60
+
+
+def file_age_check(filename, warning, critical, optional):
+    """file_age_check
+
+    Checks the age and existence of a given filename
+
+    Arguments:
+      - filename: a string, containing a full path
```

Comment:
I'm not sure if the dashes are needed or if they'll throw off RTD.  One way to test is to 
1. create a virtualenv using `requirements-docs.txt`
2. `cd docs; make html`
3. open `_build/html/scriptworker.html` in your browser, and look at the formatting of these docstrings in the docs.

If they look ok, I'm not going to push too hard.  If they look odd or jarringly different from the other docstrings, the formatting from http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html is:

```
        Args:
            param1 (str): Description of `param1`.
            param2 (:obj:`int`, optional): Description of `param2`. Multiple
                lines are supported.
            param3 (:obj:`list` of :obj:`str`): Description of `param3`.
```